plugins/ooni/main.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "OONI",
+    "name": "ooni",
```

Comment:
Got it, thanks for explaining! (As a side note, I think lowercase names are more suitable for plugins on the filesystem.)